src/main/kotlin/com/hubermjonathan/discord/common/models/Feature.kt

```diff
@@ -4,29 +4,32 @@ import com.hubermjonathan.discord.common.DiscordLogger
 import net.dv8tion.jda.api.JDA
 import java.util.Timer
 
-abstract class Feature(jda: JDA, name: String, icon: String) {
+abstract class Feature(private val jda: JDA, private val name: String, icon: String) {
     abstract val buttons: List<Button>
     abstract val commands: List<Command>
     abstract val managers: List<Manager>
     abstract val tasks: List<Task>
     protected val context = FeatureContext(jda, logger = DiscordLogger(jda, name, icon), name, icon)
 
-    fun load(jda: JDA) {
+    fun load() {
```

Comment:
I couldn't think of a a good reason why the `jda` was being passed down, particularly since it was being created before any of the features were instantiated. Probably leftover from some refactoring.